src/models/index.ts

```diff
@@ -37,6 +37,20 @@ import MsgDtagRefuseTransfer from './msg/profiles/msg_dtag_refuse_transfer';
 import MsgDtagTransferRequest from './msg/profiles/msg_dtag_transfer_request';
 import MsgSaveProfile from './msg/profiles/msg_save_profile';
 import MsgUnblockUser from './msg/profiles/msg_unblock_user';
+import MsgCreatePool from './msg/osmosis/gamm/v1beta1/msg_create_pool';
+import MsgExitPool from './msg/osmosis/gamm/v1beta1/msg_exit_pool';
+import MsgExitSwapExternAmountOut from './msg/osmosis/gamm/v1beta1/msg_exit_swap_extern_amount_out';
+import MsgExitSwapShareAmountIn from './msg/osmosis/gamm/v1beta1/msg_exit_swap_share_amount_in';
+import MsgJoinPool from './msg/osmosis/gamm/v1beta1/msg_join_pool';
+import MsgJoinSwapExternAmountIn from './msg/osmosis/gamm/v1beta1/msg_join_swap_extern_amount_in';
+import MsgJoinSwapShareAmountOut from './msg/osmosis/gamm/v1beta1/msg_join_swap_share_amount_out';
+import MsgSwapExactAmountIn from './msg/osmosis/gamm/v1beta1/msg_swap_exact_amount_in';
+import MsgSwapExactAmountOut from './msg/osmosis/gamm/v1beta1/msg_swap_exact_amount_out';
+import MsgCreateGauge from './msg/osmosis/incentives/msg_create_gauge'
+import MsgAddToGauge from './msg/osmosis/incentives/msg_add_to_gauge';
+import MsgBeginUnlocking from './msg/osmosis/lockup/msg_begin_unlocking';
+import MsgBeginUnlockingAll from './msg/osmosis/lockup/msg_begin_unlocking_all';
+import MsgLockTokens from './msg/osmosis/lockup/msg_lock_tokens';
```

Comment:
instead of `./msg/osmosis` can we just split them by modules? 
ex i could've done `./msg/desmos/profiles` but i decided to do `./msg/profiles` to follow the module pattern.

So for ex:
`msg_create_pool` just do `./msg/gamm/msg_create_pool` // don't know if we need to say its `v1beta1` 
`msg_lock_tokens` `./msg/lockup`

Reply:
Okay I updated the code and split them by modules